pkg/payload/payload_test.go

```diff
@@ -123,3 +123,52 @@ func mustRead(path string) []byte {
 	}
 	return data
 }
+
+func TestExclude(t *testing.T) {
+	tests := []struct {
+		exclude     string
+		profile     string
+		annotations map[string]interface{}
+
+		isExcluded bool
+	}{
+		{
+			exclude:     "",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+		},
+		{
+			exclude:     "identifier",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "crc",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "edge",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+		},
+		{
+			profile: "default",
+		},
+		{
+			profile: "edge",
```

Comment:
this should exclude the manifest, because the manifest is implicitly a member of the `default` profile, and not a member of the `edge` profile.

Reply:
This case ensures we don't have to duplicate all manifests for all profiles. We can probably change the enhancemen to say 
`Manifests with no explicit inclusion annotations implicitly belong to all profiles. `.

Reply:
> This case ensures we don't have to duplicate all manifests for all profiles.

David [wants][1] that legwork to be a new-profile entry barrier.

[1]: https://github.com/openshift/enhancements/pull/200#discussion_r375837903